test/network_test.rb

```diff
@@ -0,0 +1,46 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/character'
+require './lib/show'
+require './lib/network'
+require 'pry'
+
+class NetworkTest < MiniTest::Test
+
+  def setup
+    @nbc = Network.new("NBC")
+
+    @michael_knight = Character.new({name: "Michael Knight", actor: "David Hasselhoff", salary: 1_600_000})
+    @kitt = Character.new({name: "KITT", actor: "William Daniels", salary: 1_000_000})
+    @knight_rider = Show.new("Knight Rider", "Glen Larson", [@michael_knight, @kitt])
+
+    @leslie_knope = Character.new({name: "Leslie Knope", actor: "Amy Poehler", salary: 2_000_000})
+    @ron_swanson = Character.new({name: "Ron Swanson", actor: "Nick Offerman", salary: 1_400_000})
+    @parks_and_rec = Show.new("Parks and Recreation", "Michael Shur & Greg Daniels", [@leslie_knope, @ron_swanson])
+  end
+
+  def test_it_exist
+    assert_instance_of Network, @nbc
+  end
+
+  def test_network_attributes
+    assert_equal "NBC", @nbc.name
+    assert_equal [], @nbc.shows
+  end
+
+  def test_add_show
+    @nbc.add_show(@knight_rider)
+    @nbc.add_show(@parks_and_rec)
+
+    assert_equal [@knight_rider, @parks_and_rec], @nbc.shows
+  end
+
+  def test_payroll
+    skip
+    @nbc.add_show(@knight_rider)
+    @nbc.add_show(@parks_and_rec)
+    @payroll_hash = {"David Hasselhoff" => 1600000, "William Daniels" => 1000000, "Amy Poehler" => 2000000, "Nick Offerman" => 1400000}
+  
+    assert_equal @payroll, @nbc.payroll
+  end
```

Comment:
good start on this last test; looks like your assertion would be looking for `@payroll_hash` (which also doesn't need to be an instance variable) instead of `@payroll`.

looks like you're missing a test for the `highest_paid_actor` method though from the end of iteration 3